sound/soc/sof/loader.c

```diff
@@ -210,30 +211,51 @@ static int load_modules(struct snd_sof_dev *sdev, const struct firmware *fw)
 int snd_sof_load_firmware_memcpy(struct snd_sof_dev *sdev,
 				 const struct firmware *fw)
```

Comment:
Looks like you are no longer user the fw argument? Should it be removed?

Reply:
good point, let me remove it.